src/idl_gen_cpp.cpp

```diff
@@ -708,10 +709,20 @@ struct IsAlnum {
   }
 };
 
-// Iterate through all definitions we haven't generate code for (enums, structs,
-// and tables) and output them to a single file.
-std::string GenerateCPP(const Parser &parser,
-                        const std::string &file_name) {
+static std::string GeneratedFileName(const std::string &path,
```

Comment:
It hasn't be added. It currently lives at lines 897-900 in your current code base in the idl_gen_cpp.cpp file

It has just been moved before the class because it is used inside it.